app/javascript/components/modals/contact-us/styles.scss

```diff
@@ -2,86 +2,89 @@
 $mobile-padding: rem(20px);
 $desktop-padding: rem(40px);
 
-.c-form-container {
-  position: relative;
-  height: 100%;
-  width: 100%;
-  top: 0;
-  left: 0;
-  padding: 3.125rem;
-  padding-top: 2.5rem;
-  border-bottom: 66px solid transparent;
-  outline: none;
-  margin: 0 auto;
-  max-width: 700px;
-  color: #333;
-  border: none;
-  box-shadow: none;
-  background: transparent;
-  border-radius: 0;
-  font-size: 14px;
-  line-height: 1.25;
-
-  .c-button {
-    margin-bottom: 30px;
-  }
-
-  h3 {
-    font-weight: 500;
-    font-size: 1rem;
-    text-transform: uppercase;
-    color: #555;
+.c-contact-us {
+  .c-contact-form {
```

Comment:
Another thing we try to avoid is referencing component parent classes inside a css file. So in this case you are changing the components styles from the outside, but if there were multiple instances of this component rendered and you only wanted to change this instance you would have a problem. Can you try to assign a theming class here to adjust layout as desired.